man/man5/zfs-module-parameters.5

```diff
@@ -1752,14 +1769,97 @@ Default value: \fB2\fR.
 .ad
 .RS 12n
 Max concurrent I/Os per top-level vdev (mirrors or raidz arrays) allowed during
-scrub or resilver operations.
+legacy scrub or resilver operations.
 .sp
 Default value: \fB32\fR.
 .RE
 
 .sp
 .ne 2
 .na
+\fBzfs_scan_md_maxinflight\fR (int)
+.ad
+.RS 12n
+Max concurrent I/Os allowed during scrub or resilver metadata scanning phase.
+.sp
+Default value: \fB1024\fR.
```

Comment:
> When our ~1 second per txg is up

Not quite sure what you mean by this....

> how long does it take to finish these? Seems like it could be up to around 10 seconds (e.g. if we have a single top-level vdev composed of spinning disks).

I am completely open to changing the tunable defaults. These numbers seemed to give us good performance during our testing on 36-drive spinning disk pools. What do you think a better value would be?